healthcheck/settings/production.py

```diff
@@ -1,6 +1,6 @@
 from healthcheck.settings.base import *  # noqa: F403
 
-SECRET_KEY = env.str("SECRET_KEY")  # noqa: F405
+SECRET_KEY = env.str("SECRET_KEY", "replaceme")  # noqa: F405
```

Comment:
I think we should put this back as it was, we don't want to risk running a production instance without a proper secret key, we would rather have it fail to start up

Reply:
Got it, will change